commit fc415edea82df91ceecf6cb0c2e69c3cf0dc1583
Author: Xu Fasheng <fasheng.xu@gmail.com>
Date:   2015-09-08 15:25:56 +0800

    fix go template panic issue under go1.5
    
    For go1.5, double dot in template will cause
    parser error like {{..E}}.
    
    https://github.com/golang/go/commit/76ace947aeaa53e4f743f0a3bc69cea17e63580a
    
    Change-Id: If9290c5cd65de100051f30e60f104cf01d24d92c

diff --git a/golang/src/make_dbus/make_dbus.go b/golang/src/make_dbus/make_dbus.go
index 46f20c3..982e404 100644
--- a/golang/src/make_dbus/make_dbus.go
+++ b/golang/src/make_dbus/make_dbus.go
@@ -165,10 +165,10 @@ static void _bus_method_call (GDBusConnection * connection,
         {{$arg.CName}} arg{{$index}};
     {{end}}
     {{get_c_content .}}
-    {{if .Ret.Type }} {{..Ret.CName}} _c_retval ={{end}} {{gen_arg_call .}}
+    {{if .Ret.Type }} {{.Ret.CName}} _c_retval ={{end}} {{gen_arg_call .}}
 
     {{if .Ret.Type }}
-        retval = g_variant_new("({{..Ret.DName}})", _c_retval);
+        retval = g_variant_new("({{.Ret.DName}})", _c_retval);
         {{if is_string .Ret.CName}}g_free(_c_retval);{{end}}
     {{end}}
         g_dbus_method_invocation_return_value (invocation, retval);
@@ -215,7 +215,7 @@ void
 {{range .Methods}}{{if .IsSignal}}"       <signal name=\"{{.Name}}\">"
 {{range .Args}}"             <arg name=\"{{.Name}}\" type=\"{{.DName}}\" ></arg>"
 {{end}}"       </signal>"{{else}}"       <method name=\"{{.Name}}\">"
-{{if .Ret.Type}}"             <arg name=\"{{..Ret.Name}}\" type=\"{{..Ret.DName}}\" direction=\"out\"></arg>"
+{{if .Ret.Type}}"             <arg name=\"{{.Ret.Name}}\" type=\"{{.Ret.DName}}\" direction=\"out\"></arg>"
 {{end}}{{range .Args}}"             <arg name=\"{{.Name}}\" type=\"{{.DName}}\" direction=\"{{.Type}}\"></arg>"
 {{end}}"       </method>" {{end}}
 {{end}}"</interface>"
